2_base_and_sensor.ipynb

```diff
@@ -155,11 +288,22 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "台車を直進させ続けてみましょう。while文を用いることでロボットを動かし続けることができます。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's keep the move base going straight. You can keep the robot moving by using the `while` statement:"
```

Comment:
"Let's keep the moving base going forward. You can keep the robot moving by using the `while` statement:"